biz.aQute.repository/src/aQute/bnd/deployer/repository/LocalIndexedRepo.java

```diff
@@ -69,6 +105,30 @@ public synchronized void setProperties(Map<String,String> map) {
 			throw new IllegalArgumentException(String.format(
 					"Cannot create repository cache: '%s' already exists but is not directory.",
 					cacheDir.getAbsolutePath()));
+
+		setIncludePolicy(map);
```

Comment:
Why pass the whole map when only the files prop is needed? Just pass the prop value.